package.json

```diff
@@ -11,24 +11,24 @@
     "benchmark:parser": "concurrently -k -s first \"node ./examples/benchmark/parser.js\" \"autocannon -c 100 -d 30 -p 10 -i ./examples/benchmark/body.json -H \"content-type:application/jsoff\" -m POST localhost:3000/\"",
     "build:validation": "node build/build-error-serializer.js && node build/build-validation.js",
     "coverage": "npm run unit -- --coverage-report=html",
-    "coverage:ci": "c8 --reporter=lcov tap --coverage-report=html --no-browser --no-check-coverage",
-    "coverage:ci-check-coverage": "c8 check-coverage --branches 100 --functions 100 --lines 100 --statements 100",
+    "coverage:ci": "tap --coverage-report=html --coverage-report=lcov --allow-incomplete-coverage",
+    "coverage:ci-check-coverage": "tap replay",
```

Comment:
Replay commands will fail if threshold is not meet
See https://github.com/fastify/fastify/actions/runs/9362073855 for demostration